--- shermans_aquarium-2.2.0/src/Makefile.in~	2010-01-06 01:50:25.099842691 +0100
+++ shermans_aquarium-2.2.0/src/Makefile.in	2010-01-06 01:54:54.059937704 +0100
@@ -54,7 +54,8 @@
 	-D$(HOST) @USE_GNOME@ @WITH_THERMO@ @WITH_CLOCK@ @WITH_LEDS@ @WITH_MATRIX@ @GTK2@ @FULLSCREEN@ \
 	$(GTK_CFLAGS) $(GDK_PIXBUF_CFLAGS) $(@USE_GNOME_CFLAGS@) $(@USE_ORBIT_CFLAGS@) $(@USE_SDL_CFLAGS@)
 
-LDFLAGS = @LDFLAGS@ $(GTK_LIBS) $(GDK_PIXBUF_LIBS) $(@USE_GNOME_LDFLAGS@) $(@USE_SDL_LIBS@) $(SYS_SPEC_LIBS)
+LDFLAGS = @LDFLAGS@
+LIBS = @LIBS@ $(@USE_GNOME_LDFLAGS@) $(GTK_LIBS) $(GDK_PIXBUF_LIBS) $(@USE_SDL_LIBS@) $(SYS_SPEC_LIBS) -lgnomeui-2
 
 OBJS = aquarium.o @SYSLOAD_OBJ@ @KLEDS_OBJ@ draw.o wmapplet.o detectbrokenwm.o @GNOME_OBJ@ @PREF_OBJ@ @MATRIX_OBJ@ @GETOPT_OBJ@ @FULLSCREEN_OBJ@
 
@@ -62,7 +63,7 @@
 all: sherman
 
 sherman: stamp.h $(OBJS)
-	$(CC) $(LDFLAGS) -o sherman $(OBJS) 
+	$(CC) $(LDFLAGS) -o sherman $(OBJS) $(LIBS)
 
 install: sherman @INSTALL_CHOISE@
 
